workshop/6-CustomerDataPlatforms/6.1-Segment.ipynb

```diff
@@ -0,0 +1,303 @@
+    "\n",
+    "Before you set up an omnichannel personalization pipeline using Segment and AWS Personalize, let's review the relevant parts of the Retail Demo Store architecture and how it uses Segment to integrate with the machine learning campaigns created in Personalize.\n",
+    "\n",
+    "\n",
+    "![Architecture Overview](images/segment/segment_personas_architecture.png)"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "## Setup\n",
+    "\n",
+    "For this exercise, we are going to start by creating a Python data source within Segment. Log in to Segment. From the Segment app home screen, click the Add Source button.\n",
+    "\n",
+    "![Add a source in Segment](images/segment/add_source.png)\n",
+    "\n",
+    "Find the Python source and click Add Source.\n",
+    "\n",
+    "![Add a Python source in Segment](images/segment/add_source_2.png)\n",
+    "\n",
+    "Once you’ve created and named the new Python Source, you will be presented with a write key:\n",
+    "\n",
+    "![Add a Python source in Segment](images/segment/add_source_3.png)\n",
+    "\n",
+    "Copy your write key into the string below, and run the next cell:"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enter your Segment write key here\n",
+    "\n",
+    "segment_write_key = 'THIS_WILL_BE_YOUR_WRITE_KEY'\n"
+   ]
+  },
+  {
+   "source": [
+    "Now that we’ve configured our data collection infrastructure, we will configure Segment to allow our Python Personalize Source to feed data to Personas. Personas is a powerful personalization platform within Segment that enables you to create omni-channel customer profiles, to build and enrich audiences, and to synchronize user data and audiences across your stack. \n",
+    "\n",
+    "Navigate to Personas, and then from the Personas Settings Page (personas/settings/sources), click the Connect Sources button. \n",
+    "\n",
+    "![Add a Python source in Segment](images/segment/add_source_4.png)\n",
+    "\n",
+    "Then choose the Python Personalise source and click the Connect Sources button.\n",
+    "\n",
+    "![Add a Python source in Segment](images/segment/add_source_5.png)\n",
+    "\n",
+    "## Pull data from Amazon Personalize and send it to Segment\n",
+    "\n",
+    "Now that Segment is configured, let’s pull data from our Personalize demo and send it to Segment. First, you will need to get the name of the load balancer for the Retail Demo Store recommendations service you deployed at the beginning of the Retail Demo Store workshop.\n",
+    "The following code will query the AWS account you are using for this workshop, get a list of load balancers, and find the one that belongs to the Retail Demo Store and has the tag recommendations. This URL will be stored in the recommendations_elb_domain_name variable after you run this code."
```

Comment:
Rather than referring to "data" in the title and paragraph, I suggest changing it to "recommendations". Data just seems too generic.